Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.    vm_nic_dict = dict()
</pre><p>Pylint and other places warn on this - they say use {} instead. Just FYI :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>updated!</p>